### ISSUE TITLE
Migrate users to use the django built in active flag instead of the custom approved flag in UserProfile

### ISSUE THREAD

**Issue by CarvellWakeman** [osuosl/streamwebs#614]

**Comment by hertelc** [comment]
@CarvellWakeman done?

**Comment by CarvellWakeman** [comment]
Nope, I wanted to wait on this because it's technically not necessary, but should have been done from the get go. It might enable some security features for us because the django built in is_active flag some some stuff for you automatically. We are using a custom column for our userProfile object called approved.

There was probably some reason for the decision, but I have no idea what it was.
